datamonster_api/lib/company.py

```diff
@@ -14,13 +14,22 @@ def __init__(self, id_, ticker, name, uri, dm):
         self.dm = dm
 
     def get_details(self):
+        """Get details (metadata) for this company
+        """
         return self.dm.get_company_details(self.id)
 
     @property
     def datasources(self):
-        """Get the data sources for this company"""
-
+        """Get the data sources for this company
+        """
         if not hasattr(self, '_datasources'):
             self._datasources = self.dm.get_datasources(company=self)
 
         return self._datasources
+
+    @property
+    def pk(self):
```

Comment:
Why have this?  It's just the id (which is more human readable).

Reply:
`id` is a string; in the case of `Company`s, it's str(some_pk); here's that pk.

Reply:
I think so. It's a convenience/quality of life thing -- used 2x already. The duck-type of `Company.id` is `int or str` -- actually, `int or six.text_type` -- whereas anyone naturally thinks of it as `int`; it's helpful to have a way to obtain it as an int. Otherwise, all users are condemned to writing `int(company.id)` regularly.